src/index.js

```diff
@@ -4,6 +4,21 @@ import bindActionCreators from './utils/bindActionCreators'
 import applyMiddleware from './utils/applyMiddleware'
 import compose from './utils/compose'
 
+/*
+* create a function so that we can check if the function name has been atlered by minification
+* if the function has been minified and NODE_ENV !== 'production', warn the user
+*/
+function isCrushed() {}
+
+if (isCrushed.name !== 'isCrushed' && process.env.NODE_ENV !== 'production') {
+  var warningMessage = 'You are utilzing minified code outside of NODE_ENV === \'production\'. ' + 
```

Comment:
We're only using `warningMessage` once, let's put it right inside the `console.warn` call?